src/igvfd/schemas/changelogs/treatment.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`treatment.json`*
 
+### Schema version 4
+* Add `lab` and `award`.
+* Add `depletion`. It is `False` by default. If it is `True`, `amount` and `amount_units` should not be specified.
```

Comment:
I think change log should be very terse.  No need to explain defaults, or logic of other properties.